### PR TITLE
ci: pull trivy DB from public AWS ECR

### PR DESCRIPTION
**Pull request by DavSanchez**

### CODE REVIEW

File: .github/workflows/download_trivy_db.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Update Trivy Cache
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # Update daily at 00:00 - before scheduled trivy scan
+  workflow_dispatch:  # Allow manual triggering
```
Comment (paologallinaharbur) [inline .github/workflows/download_trivy_db.yaml:6]: if you want to make it reusable you should add 
```yaml
on:
  workflow_call:
    inputs: [...] # not sure if these are needed
```

Comment (sigilioso) [reply .github/workflows/download_trivy_db.yaml:6]: 🤔 If we make this reusable, we we'll also need to open 20+ PRs for all the repositories using them, right? I guess we cannot have a cross-repo cache...

Comment (DavSanchez) [reply .github/workflows/download_trivy_db.yaml:6]: I guess these PRs need to be open as well to reference this one, yes 🫠

Comment (DavSanchez) [reply .github/workflows/download_trivy_db.yaml:6]: Added in https://github.com/newrelic/coreint-automation/pull/84/commits/1f03e76f865749e8f92eee74522f9831ef0b6690

**Comment by DavSanchez** [comment]
I checked [another workaround](https://github.com/aquasecurity/trivy-action/issues/389#issuecomment-2468215171) that does not require opening PRs in all integrations. Added in commit https://github.com/newrelic/coreint-automation/pull/84/commits/bb741a1cf20f2ab4031fc702eb6042cab9952ece.

**Review by sigilioso — APPROVED** [review]
LGTM! 🚀 

We'll need to move `v3` tag when this is merged

**Pull request merged**